File: examples/museum/oseg_generated/python/DeleteSpecialEvent_default_example.py
from datetime import date, datetime
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        api.EventsApi(api_client).delete_special_event(
            event_id="dad4bce8-f5cb-4078-a211-995864315e39",
        )
    except ApiException as e:
        print("Exception when calling Events#delete_special_event: %s\n" % e)

File: examples/museum/oseg_generated/php/DeleteSpecialEventDefaultExample.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPIMuseum\Client\Configuration::getDefaultConfiguration();

try {
    (new OpenAPIMuseum\Client\Api\EventsApi(config: $config))->deleteSpecialEvent(
        event_id: "dad4bce8-f5cb-4078-a211-995864315e39",
    );
} catch (OpenAPIMuseum\Client\ApiException $e) {
    echo 'Exception when calling Events#deleteSpecialEvent: '
        . print_r($e->getResponseObject());
}
